tests/mini_stdlib/test0/fingerprint.out

```diff
@@ -1 +1 @@
-9.4876E-02 0.0000E+00 6.9364E-02 3.9104E+00 4.3959E+00 3.9078E+00 3.0695E-01 0.0000E+00 7.5145E-02 3.9042E+00 4.3903E+00 3.9026E+00
```

Comment:
Why did the test results changed?  Is it because the data order changed with loader changes?

Reply:
Yes we changed fingerpints in much earlier commit: https://github.com/IBM/graph2tac/pull/86/commits/e3116eee0c5049d1d5a69e4dffc42fdf793ca9a4

The purpose was to align tactic hash indexing between the two loaders. 

The original loader was indexing tactic hashes by the lexicographical order (by value of the tactic hash).

Mirek's loader was using the chronological order of walking through the dataset, and I switched to the chronological order as well so that it would be easier for Mirek to compare the output of the two loaders. 

With that change the two loaders seem to use the same ordering of tactic hashes.